examples/etcd/etcd-in-cluster/ocp/README.md

```diff
@@ -0,0 +1,274 @@
+  ----    ------           ----  ----                 -------
+  Normal  Started Action   3m    Kanister Controller  Executing action backup
+  Normal  Started Phase    3m    Kanister Controller  Executing phase takeSnapshot
+  Normal  Ended Phase      3m    Kanister Controller  Completed phase takeSnapshot
+  Normal  Started Phase    3m    Kanister Controller  Executing phase uploadSnapshot
+  Normal  Ended Phase      2m    Kanister Controller  Completed phase uploadSnapshot
+  Normal  Started Phase    2m    Kanister Controller  Executing phase removeSnapshot
+  Normal  Ended Phase      2m    Kanister Controller  Completed phase removeSnapshot
+  Normal  Update Complete  2m    Kanister Controller  Updated ActionSet 'backup-4f6jn' Status->complete
+```
+
+## Imitate Disaster
+
+After the bakcup has successfully been taken, let's go ahead and delete the dummy namespace that we created to imitate the disaster
+
+```
+root@workmachine:/repo# oc delete ns nginx
+namespace "nginx" deleted
+
+root@workmachine:/repo# oc get all -n nginx
+No resources found.
+```
+
+
+## Restore ETCD cluster
+
+To restore the ETCD cluster we can follow the [documentation](https://docs.openshift.com/container-platform/4.5/backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.html) that is provided the OpenShift team.
+But we will have to make some modification in the restore script (`cluster-restore.sh`) because default
+restore script expects the static pods manifest as well and in our case we didn't backup the satic pod manifests.
+
+You can follow the steps that are mentioned below along with the documentation that is mentioned above, most of the steps that are mentioned here are either directly taken from the documentation or are modified version of it. Among all the running leader nodes choose one node to be the restore node, make sure you have SSH connectivity to all of the leader nodes including the one that you have chosen to be restore node.
+
+You will have to have a command line utiltiy that can be used to download the ETCD snapshot that we have taken in the eariler step, that will depend on the object
+storage that you used. For example if you used the object storage to be AWS S3, you will need `awc` cli to download the etcd snapshot. Once you have the CLI installed
+on the restore host, below steps can be followed to restore ETCD:
+
+- Download the ETCD snapshot on the restore host using the aws cli on a specific path let's say `/var/home/core/etcd-backup`
+- Stop the static pods from all other leader hosts (not the recovery host) by copying the manifests out of the static pod path dir, i.e. `/etc/kubernetes/manifests`
+
+  ```
+  # move etcd pod manifest
+  sudo mv /etc/kubernetes/manifests/etcd-pod.yaml /tmp
+
+  # make sure etcd pod has been stopped
+  sudo crictl ps | grep etcd
+
+  # move api server pod
+  sudo mv /etc/kubernetes/manifests/kube-apiserver-pod.yaml /tmp
+  ```
+
+- move the etcd data dir to a different location
```

Comment:
```suggestion
- move the etcd data dir to a different location.
```

Reply:
let's not have pull stop in the unordered list.

Reply:
resolving this.